cumulus_library/study_manifest.py

```diff
@@ -117,13 +133,20 @@ def get_export_table_list(self) -> list[str] | None:
         :returns: An array of tables to export from the manifest, or None if not found.
         """
         export_config = self._study_config.get("export_config", {})
-        export_table_list = export_config.get("export_list", []) or []
-        for table in export_table_list:
-            if not table.startswith(f"{self.get_study_prefix()}__"):
+        export_table_list = []
+        for table in export_config.get("export_list") or []:
+            if table.startswith(f"{self.get_study_prefix()}__"):
+                export_table_list.append(table)
+            elif "__" in table:  # has a prefix, just the wrong one
                 raise errors.StudyManifestParsingError(
                     f"{table} in export list does not start with prefix "
                     f"{self.get_study_prefix()}__ - check your manifest file."
                 )
+            else:
+                # Add the prefix for them (helpful in dynamic prefix cases where the prefix
+                # is not known ahead of time)
+                export_table_list.append(f"{self.get_study_prefix()}__{table}")
```

Comment:
This is a notable change - this allows a study author to do:
```
[export_config]
export_tables = [
  "meta_version",
]
```

without a prefix.

Reply:
Interesting - I don't think we should encourage that, though. We don't have to change the logic, but let's just not recommend it.